src/tools/launcher/bash_launcher.cc

```diff
@@ -30,6 +30,12 @@ static constexpr const char* BASH_BIN_PATH = "bash_bin_path";
 
 ExitCode BashBinaryLauncher::Launch() {
   wstring bash_binary = this->GetLaunchInfoByKey(BASH_BIN_PATH);
+
+  if (GetBinaryPathWithoutExtension(bash_binary) == L"bash") {
```

Comment:
Just checking: `bash_binary` can either be an absolute Bash path (from `BAZEL_SH` or `--shell_executable`), or (theoretically perhaps) a bash toolchain's runfiles-relative path. Correct?

And it always ends with "bash" or "bash.exe" (so just "bash" without the extension), right?

Any other option I'm missing?

Reply:
Everything you said I believe is true, I don't think there's any other option.

Reply:
Thanks for confirming that!

Reply:
Could you please add a comment about this?